policyengine-client/src/landing.jsx

```diff
@@ -37,60 +34,54 @@ function UsageBox(props) {
 
     // Displays the title anchored to the left (and big) and the description anchored to the right (and small)
 
-    return <Row style={{marginTop: 20, marginBottom: 20}}>
-        <Col xs={12} md={6}>
-            <h3 style={{marginBottom: 0}}>{props.title}</h3>
-        </Col>
-        <Col xs={12} md={6}>
-            <h6 style={{marginTop: 0}}>{props.description}</h6>
-        </Col>
+    return <Row style={{ marginTop: 20, marginBottom: 20 }}>
+        <h3 style={{ marginBottom: 0 }}>{props.title}</h3>
+        <br /><br />
+        <h5 style={{ marginTop: 0 }}>{props.description}</h5>
     </Row>
 
 
 }
 
 function UsageExplanations(props) {
     return <>
-        <UsageBox title="Citizens" description="Check your eligibility for government benefits and programs. Simulate how a change in requirements could affect your household." />
-        <UsageBox title="Think tanks" description="Simulate tax-benefit reforms on the UK economy and individual households to better understand and recommend policy changes." />
-        <UsageBox title="Parties and campaigns" description="Estimate the economic outcomes of your policy proposals. Produce evidence-based talking points and refute speculative criticism of your platform." />
-        <UsageBox title="Developers" description={<>Allow your users to check their benefits eligibility with our API. <a href="https://github.com/PolicyEngine">Contribute to our work on GitHub.</a></>} />
+        <UsageBox title="The public" description="Estimate your taxes and government benefits, under current law and potential policy changes" />
```

Comment:
Could we add a centered dot between each of these? Just to separate them, as on mobile at first glance it kind of looks like they might be one multi-line point.

Reply:
Sorry, my bad - I meant the taglines at the top (estimate your taxes and benefits ... ). Though actually it's pretty marginal